libqtile/scripts/migrate.py

```diff
@@ -130,6 +138,8 @@ def windowtogroup_groupName_argument(funcname, query):  # noqa: N802
     tile_master_windows_rename,
     threaded_poll_text_rename,
     pacman_to_checkupdates,
+    border_to_border_color,
+    borderwidth_to_border_width,
```

Comment:
If we're adding migrate scripts, we should test them.

Have a look at https://github.com/qtile/qtile/blob/master/test/test_migrate.py for examples.